.rubocop.yml

```diff
@@ -41,6 +41,8 @@ Metrics/MethodLength:
   CountComments: false
   Max: 20
   Severity: error
+  Exclude:
+    - lib/jekyll/command.rb
```

Comment:
This one was tricky for me. The offending method is here:

```ruby
      def add_build_options(c)
        c.option "config", "--config CONFIG_FILE[,CONFIG_FILE2,...]",
          Array, "Custom configuration file"
        c.option "destination", "-d", "--destination DESTINATION",
          "The current folder will be generated into DESTINATION"
        c.option "source", "-s", "--source SOURCE", "Custom source directory"
        c.option "future", "--future", "Publishes posts with a future date"
        c.option "limit_posts", "--limit_posts MAX_POSTS", Integer,
          "Limits the number of posts to parse and publish"
        c.option "watch", "-w", "--[no-]watch", "Watch for changes and rebuild"
        c.option "baseurl", "-b", "--baseurl URL",
          "Serve the website from the given base URL"
        c.option "force_polling", "--force_polling", "Force watch to use polling"
        c.option "lsi", "--lsi", "Use LSI for improved related posts"
        c.option "show_drafts", "-D", "--drafts", "Render posts in the _drafts folder"
        c.option "unpublished", "--unpublished",
          "Render posts that were marked as unpublished"
        c.option "quiet", "-q", "--quiet", "Silence output."
        c.option "verbose", "-V", "--verbose", "Print verbose output."
        c.option "incremental", "-I", "--incremental", "Enable incremental rebuild."
        c.option "strict_front_matter", "--strict_front_matter",
          "Fail if errors are present in front matter"
      end
```

We get dinged b/c it's over 20 lines. It's right at 21 lines. Coalescing any of the options that cover multiple lines brings the line length over 90 characters. Breaking it into `add_build_options_a_through_m` and `add_build_options_n_through_z` is the next best thing I could think of, which would be ridiculous of course.

I'm welcoming thoughts on this b/c I agree that changing the cop is cheating.